src/elm/Page/Community/Invite.elm

```diff
@@ -64,54 +47,112 @@ type ModalStatus
     | Open
 
 
+
+-- MODEL
+
+
+type alias Model =
+    { pageStatus : PageStatus
+    , confirmationModalStatus : ModalStatus
+    , invitationId : InvitationId
+    }
+
+
+
+-- INIT
+
+
+initModel : String -> Model
+initModel invitationId =
+    { pageStatus = Loading
+    , confirmationModalStatus = Closed
+    , invitationId = invitationId
+    }
+
+
+init : Session -> String -> ( Model, Cmd Msg )
+init session invitationId =
+    ( initModel invitationId
+    , Api.Graphql.query
+        (toShared session)
+        (Community.inviteQuery invitationId)
+        CompletedLoad
+    )
+
+
+
+-- VIEW
+
+
 view : Session -> Model -> { title : String, content : Html Msg }
 view session model =
     let
         shared =
             toShared session
 
+        { t } =
+            shared.translators
+
         title =
-            case model.status of
+            case model.pageStatus of
                 Loaded invite ->
                     let
                         inviter =
                             invite.creator.userName
-                                |> Maybe.withDefault (Eos.nameToString invite.creator.account)
+                                |> Maybe.withDefault (nameToString invite.creator.account)
                     in
                     inviter
                         ++ " "
-                        ++ t shared.translations "community.invitation.title"
+                        ++ t "community.invitation.title"
                         ++ " "
                         ++ invite.community.title
 
                 _ ->
                     ""
 
         content =
-            div [ class "flex flex-col min-h-screen" ]
-                [ div [ class "flex-grow" ]
-                    [ case model.status of
-                        Loading ->
-                            div [] []
-
-                        NotFound ->
-                            div [] []
-
-                        Failed e ->
-                            Page.fullPageGraphQLError (t shared.translations "") e
-
-                        Loaded invite ->
-                            div []
-                                [ viewHeader
-                                , viewContent shared invite model.invitationId
-                                , viewModal shared model model.invitationId
-                                ]
-
-                        Error e ->
-                            Page.fullPageError (t shared.translations "") e
-                    ]
-                , viewFooter session
-                ]
+            case model.pageStatus of
+                Loading ->
+                    div [] []
```

Comment:
since we are modifying this, maybe we can just use 

```
text ""
```

Reply:
Done!